.github/workflows/bats.yaml

```diff
@@ -73,8 +73,12 @@ jobs:
     - name: Consolidate logs
       if: ${{ success() || failure() }}
```

Comment:
Is that meant to be `if: always()`? Or is that meant to be `if: !cancelled()`?
(That's not changed in this PR, so if you want to keep it then it's fine)